draft-rokui-ccamp-actn-wdm-pluggable-modelling.md

```diff
@@ -568,36 +524,32 @@ A variety of performance monitoring metrics, including minimum, maximum, average
-
-Note that sometimes these thresholds are configurable and sometime they are hard-coded. It is also possible that a vendor can support a sub-set and super-set of monitored attributes (for super-set they need to augment the yang model). 
-
-
-
-
 ~~~~
-         Supported-Collection-and-Threshold-Profile (SCTP)
+         Supported-Collection-and-Threshold-Group (SCTG)
  |----------------------------------------------------------------|
- |   SCTP-Type-1:                                                 |
+ |   SCTG-Type-1:                                                 |
  |     Collection: current, average, min, max                     |
  |     Configured Threshold: warning, minor, major, critical      |
  |                                                                |
- |   SCTP-Type-2:                                                 |
+ |   SCTG-Type-2:                                                 |
  |     Collection: current                                        |
  |     Configured Threshold: warning, minor, major, critical      |
  |                                                                |
- |   SCTP-Type-3:                                                 |
+ |   SCTG-Type-3:                                                 |
  |     Collection: current, average, min, max                     |
  |   ...                                                          |
- |   SCTP-Type-n:                                                 |
- |----------------------------------------------------------------|
-    // Note: These are just a few examples. More SCTP profile 
-    //       can be defined
+ |   SCTG-Type-n:                                                 |
+ |------------------------------------------------G---------------|
+    // Note: 
+    // These are just a few examples. More SCTG can be defined
 ~~~~
-{: #figure-plug-threshold-definition title="Coherent pluggable Collection and Threshold Profile Definition"}
+{: #figure-plug-threshold-definition title="Coherent pluggable Collection and Threshold Group Definition"}
 
+As mentioned previously, the TCA might be supported for a subset of coherent pluggable monitored attributes. Since it is possible that the coherent pluggable has different capabilities to raise threshold for different monitored attributes, to provide a general solution for threshold definition on coherent pluggable monitored attributes, this draft introduces the concept of "Supported Collection and Threshold Group (SCTG)" shown in {{figure-plug-threshold-definition}} which defines the configurable threshold values and collection types (i.e., the collection of current value, average value, min/max value are supported). The SCTG types will be used in list of coherent pluggables Sheet.
```

Comment:
We should draw clearer distinction between gap analysis vs. modeling proposals. If there is a gap identified related to TCA, it has to be clear which is the reference for the gap. 
if there is a proposed solution for the gap it is better to discuss this in a separate draft.